src/pages/Profile.js

```diff
@@ -207,17 +213,25 @@ const FenceLink = ({ provider: { key, name } }) => {
     div({ style: styles.form.title }, [name]),
     Utils.cond(
       [isBusy, () => div([spinner(), 'Loading account status...'])],
-      [!username, () => h(FrameworkServiceLink, { linkText: 'Log-In to Framework Services to link your account', provider: key, redirectUrl })],
-      () => div({ style: { display: 'flex', flexDirection: 'column', width: '33rem' } }, [
-        div({ style: { display: 'flex' } }, [
-          div({ style: { flex: 1 } }, ['Username:']),
-          div({ style: { flex: 2 } }, [username])
-        ]),
-        div({ style: { display: 'flex' } }, [
-          div({ style: { flex: 1 } }, ['Link Expiration:']),
-          div({ style: { flex: 2 } }, [Utils.makeCompleteDate(expireTime)])
+      [!username, () => div({ style: styles.identityLine },
+        [h(FrameworkServiceLink, { linkText: 'Log in to link your account', provider: key, redirectUrl })]
+      )],
+      () => div([
+        div({ style: { display: 'flex', flexDirection: 'column', width: '33rem' } }, [
+          div({ style: styles.identityLine }, [
+            div({ style: { flex: 1 } }, ['Username:']),
+            div({ style: { flex: 2 } }, [username])
+          ]),
+          div({ style: styles.identityLine }, [
+            div({ style: { flex: 1 } }, ['Link Expiration:']),
+            div({ style: { flex: 2 } }, [Utils.makeCompleteDate(expireTime)])
+          ])
         ]),
-        h(FrameworkServiceLink, { linkText: 'Log-In to Framework Services to re-link your account', provider: key, redirectUrl })
+        div({ style: styles.identityLine }, [
```

Comment:
If this `div` was inside the flex column just above, then we wouldn't need the additional `div` wrapper around everything. In general, we should strive for as few wrappers as possible to achieve the desired layout.

Reply:
Removed a few extra divs here, which I think is what you mean. I can't see a way to remove the flex:1 and flex:2 divs without losing the formatting, but if there is a way (or if I messed up the formatting in a way I can't see) let me know!